app/components/channel_list/index.tsx

```diff
@@ -1,20 +1,89 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
-import withObservables from '@nozbe/with-observables';
+import React, {useEffect, useState} from 'react';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import Animated, {useAnimatedStyle, useSharedValue, withTiming} from 'react-native-reanimated';
 
-import {MM_TABLES} from '@constants/database';
-import MyTeamModel from '@typings/database/models/servers/my_team';
+import {TABLET_SIDEBAR_WIDTH, TEAM_SIDEBAR_WIDTH} from '@constants/view';
+import {useTheme} from '@context/theme';
+import {makeStyleSheetFromTheme} from '@utils/theme';
 
-import ChannelsList from './channel_list';
+import Categories from './categories';
+import ChannelListHeader from './header';
+import LoadingError from './loading_error';
+import SearchField from './search';
 
-import type {WithDatabaseArgs} from '@typings/database/database';
+// import Loading from '@components/loading';
```

Comment:
Any reason to keep this comment here?

Reply:
I kept this cause this is for @shazm to remove with his other PR